dataactbroker/scripts/setupEmails.py

```diff
@@ -68,6 +69,10 @@ def setupEmails():
     template = "Your account has been unlocked and requires your password to be reset. Please click the following link <a href='[URL]'>here</a> to start the processs. The link will expire in 24 hours. <br/> <br/> If you did not request your account to be unlocked, please notify the DATA Act Broker Helpdesk (DATABroker@fiscal.treasury.gov) <br /><br />DATA Act Broker Helpdesk<br /><br />DATABroker@fiscal.treasury.gov"
     userDb.loadEmailTemplate("DATA Act Broker - Password Reset", template, "unlock_account")
 
+    #Submission Review
+    template = "[REV_USER_NAME] has shared a DATA Act broker submission with you. Click <a href='[REV_URL]'>here</a> to review their submission. For questions or comments, please email the DATA Act Broker Helpdesk (DATABroker@fiscal.treasury.gov)."
+    userDb.loadEmailTemplate("DATA Act Broker - Notification", template, "review_submission")
```

Comment:
Does it make sense to have a less generic subject line here? Something like "DATA Act Broker - Submission Ready for Review"? Better yet, maybe our content folks have an idea. Have they looked at the text for this new notification?